pepper-apis/config/squid-docker-compose.yaml

```diff
@@ -0,0 +1,7 @@
+squid:
+  image: datadog/squid:latest
+  volumes:
+    - /app/squid.conf:/etc/squid/squid.conf
+    - /var/log/squid:/var/log/squid
+  ports:
+    - "3128:3128"
```

Comment:
We should probably bind this to the internal interface- we don't want this listening on the WAN interfaces.

Reply:
I confess I don't entirely understand what that means. Any advice on how to do that?

Reply:
By default, Docker binds to `0.0.0.0`, which means `listen on all up interfaces`. Since this machine is going to have at least 2 interfaces- the external WAN interface facing the world, and the internal google interface (the one with a 10-block address), by default it will listen for connections on both.

The syntax for changing this is `IP::EXT_PORT` or `IP:INT_PORT:EXT_PORT`. The challenge is that we'll need to know the internal IP of the instance when we bring the docker containers up, and inject it into the config. Given the changes to the allowed-clients in the conf above, we can rank this as a `nice-to-have`. At first glance of the docs, it doesn't seem like it'll be a quick change.

Reply:
Okay, leaving as-is.